Zappy_IA_Report/DTC/Code/protocole.py

```
@@ -0,0 +1,210 @@
+        handle_commande(client, command, message)
+
+        if command != "Dead":
+            ml_agent.strategy(client)
+
+
+def execute_command(client, commande, args) -> None:
+    global allowed_commands
+
+    if commande not in allowed_commands:
+        raise ValueError(f"Commande '{commande}' non autorisée")
+    if commande == "Broadcast" or commande == "Set" or commande == "Take":
```

Pylint (3.3.7)

[refactor] Consider merging these comparisons with 'in' by using 'commande in ('Broadcast', 'Set', 'Take')'. Use a set instead if elements are hashable.

(R1714)